spec/models/solr_document_spec.rb

```diff
@@ -185,10 +185,6 @@
       it 'returns a ctx with a format book' do
         expect(solr_document.to_ctx(solr_document['format']).to_hash['rft.genre']).to eq('book')
       end
-
```

Comment:
We excluded titles because it caused trouble in umlaut.  We no longer use umlaut, so we can include titles in OpenURL again.